README.md

```diff
@@ -1,2 +1,22 @@
 # zap-cli
-A Node.js CLI command for searching for, or opening a YouTube video with Zen Audio Player.
+A Node.js CLI tool for searching for, or opening a YouTube video with [Zen Audio Player](https://github.com/zen-audio-player/zen-audio-player.github.io). To use the zap-cli command-line tool, clone the repository and run a global npm install inside of it.
+
+```
+git clone https://github.com/zen-audio-player/zap-cli && cd zap-cli
```

Comment:
Let's do 1 command per line, I love using `&&` but it doesn't belong in a README